### ISSUE TITLE
Update: Make modal Component mobile responsive

### ISSUE THREAD

**Issue by Arunkumar-R-R** [Arunkumar-R-R/react-comment-section#24]
Modal should be accessible and responsive
For accessibility reasons, the modal should be positioned at bottom of the mobile screen
